### ISSUE TITLE
Undefined property: stdClass::$id

### ISSUE THREAD

**Issue by PulseRetail** [gantry/gantry5#3165]
How do I resolve this issue:  Warning: Undefined property: stdClass::$id in ... /plugins/system/gantry/gantry.php on line 234?

Everything was fine until I tried to install a Gantry 5 template! Your help would be greatly appreciated.

using Joomla 3.9.x with PHP which hasn't changed in a very long time.

**Comment by PulseRetail** [comment]
Thank you for your reply. I installed the latest G5 Framework as well as a G5 template from Rockettheme. I did clear the cache several times with no luck. Cache setting is set to “file”.

**Comment by blizam** [comment]
Do you have a different template set as the default style in Joomla? I think the error is related to Gantry 5+G5 Theme not being set as Joomla's default Style/Template... 

Are you able to navigate to Components > Gantry 5 Themes > Default Theme?

**Comment by blizam** [comment]
Ultimately it's just a PHP Warning, not an Error that will crash the site; so if you're trying to build out a G5 theme alongside another currently running Joomla theme you should be able to suppress it by setting Error Report to None in your Joomla global config.

**Comment by PulseRetail** [comment]
The G5 template is not the default. The current template is G4. So if I make the G5 template the default this should correct this?

**Comment by PulseRetail** [comment]
Got it – yes, I did just realize this and can at least avoid the message being visible while I build this out.

Thank you for your help!

**Comment by blizam** [comment]
Yes, check this note from rockettheme.com for guidance about building out a G5 theme on an existing site:

https://rockettheme.com/forum/gantry-solved-threads/264325-solved-gantry-4-gantry-5

> You can create a hidden menu in Joomla (so essentially new menu in Joomla admin) and after installing a Gantry 5 template, assign the template outlines to your hidden menu items and work on the site from there. The nice thing about Gantry 5 is that you can easily create module positions so you can add those fairly easily (you can even load Joomla - Gantry 4 positions via the[ Load button](http://docs.gantry.org/gantry5/configure/layout-manager#using-the-load-button) ).
